Lib/_pydatetime.py

```diff
@@ -404,7 +404,7 @@ def _parse_isoformat_date(dtstr):
 
 
 def _parse_hh_mm_ss_ff(tstr):
-    # Parses things of the form HH[:?MM[:?SS[{.,}fff[fff]]]]
+    # Parses things of the form HH[:?MM[:?SS[{.,:}fff[fff]]]]
```

Comment:
Er, wait, why is this changed? I don''t think this is allowed by the ISO 8601 spec.

Reply:
I added next feature:
>  - [x] `_pydatetime` implementation also support `:` as microsecond separator like in `_datetime` implementation (I didn't find this in documentation)

so i fixed the docstring for this function. Should I have removed support `:` as microsec separator from `_datetime` implementation?

Reply:
p.s.: python implementation does not allow `:` as a sparator but c implementation allows

Reply:
Yes I missed this on my first look, but it is incorrect. ISO8601 does not allow a colon as the separator for the fractional portion of the time (according to Wikipedia, I can check my copy of the actual spec myself later).

If you want we can fix this in the C code later, but we definitely shouldn't be adding support for microseconds into the Python implementation.

Reply:
ok, i'll fix it later in current PR (I will disallow `:` as a separator for both implementations)